lib/devices/android/uiautomator.js

```diff
@@ -21,6 +21,7 @@ var UiAutomator = function (adb, opts) {
   this.systemPort = opts.systemPort;
   this.resendLastCommand = function () {};
   this.appPackage = opts.appPackage;
+  this.disableAndroidWatchers = opts.disableAndroidWatchers || false;
```

Comment:
this could be just `this.disableAndroidWatchers = !!opts.disableAndroidWatchers` to coerce into boolean format